lib/tasks/cleanup.rake

```diff
@@ -46,7 +51,7 @@ namespace :abbyy do
 
       num_deleted += 1
       puts "Deleting empty ABBYY output directory #{dir}"
-      FileUtils.rm_rf(dir)
+      should_perform_deletions ? FileUtils.rm_rf(dir) : puts("(dry run) would have deleted #{dir}")
```

Comment:
maybe overkill, but thought it might be nice to see what would actually get deleted, since this rake task is likely to get used infrequently to clean up after code that had unexpected bugs (whether still in development or inadvertently released to prod).

Reply:
also, if people want to leave this mode in, i'm happy to mention it in the wiki entry that mentions the cleanup task.

Reply:
wiki entry tweaked: https://github.com/sul-dlss/common-accessioning/wiki/Text-Extraction-Workflows#cleanup